client/authority-discovery/src/lib.rs

```diff
@@ -211,10 +212,23 @@ where
 			.encode(&mut serialized_addresses)
 			.map_err(Error::EncodingProto)?;
 
-		for key in self.get_priv_keys_within_authority_set()?.into_iter() {
-			let signature = key.sign(&serialized_addresses);
+		let keys: Vec<CryptoTypePublicPair> = self.get_own_public_keys_within_authority_set()?
+			.into_iter()
+			.map(|k| (sr25519::SR25519_CRYPTO_ID, k.to_raw_vec()))
+			.collect();
+
+		let signatures = self.key_store
+							 .read()
```

Comment:
Whitespaces?

Reply:
Not sure what you mean here... the editor as well as github show the func calls vertically aligned. Also, the preceding indentation is all tabs except for 1 space which i assume is to fix the vertical alignment.

Reply:
Removed the whitespace anyway